x/reward/abci.go

```diff
@@ -55,16 +55,9 @@ func excludeJailedOrTombstoned(ctx sdk.Context, slasher types.Slasher, snapshott
 		return slasher.IsTombstoned(ctx, consAdd)
 	}
 
-	isProxyActive := func(v snapshot.ValidatorI) bool {
-		_, isActive := snapshotter.GetProxy(ctx, v.GetOperator())
-
-		return isActive
-	}
-
```

Comment:
if you take this out here snapshots will be created that include disabled proxies

Reply:
multisig/evm modules use there own version of this function